nautobot/docs/user-guides/getting-started-nautobot-ui.md

```diff
@@ -0,0 +1,606 @@
+# Getting Nautobot Up and Running in the Web UI
+
+## Introduction and Scope
+
+The audience for this user guide is a user new to Nautobot. 
+It is intended to assist the user with:
+* Understanding the prerequisites to adding a new Device
+* Creating the necessary prerequisite objects to add a new Device
+* Adding new Devices
+* Adding and understanding Regions
+* Adding and Understanding Platforms
+* Adding and Understanding Tenants and Tenant Groups
+* Adding Interfaces to a Device
+* Adding VLANs and Understanding VLAN Groups
+* Understanding IP Address Management (IPAM) in Nautobot
+* How to navigate to the object you are interested in
+* How to use the Search Bar to find the object you are interested in
+
+## Requirements
+
+1. A functional Nautobot installation
+
+## Creating Devices in Nautobot
+
+A network Device in Nautobot has a few required attributes:
+* A Device Role
+* A Device Type
+  * A Device Type requires a Manufacturer
+* A Site
+
+Looking at the list above, there are four objects in Nautobot that must be present prior to creating a related Device.
+The following sections will guide you through how to create each object type.
+
+### Creating a Device Role
+
+To create a new Device, you will need an existing Device Role or need to create a new Device Role instance.
+
+To create a new Device Role:
+1. Click on **Devices** in the top navigation menu
```

Comment:
Please add a line break before all lists. It is preventing this list from rendering correctly.